tests/integration/tier1/destructive/one-kernel-scenario/test_one_kernel_scenario.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+import pytest
+
+from conftest import SYSTEM_RELEASE_ENV
+
+
+@pytest.fixture(scope="function")
+def one_kernel(shell):
+    if os.environ["TMT_REBOOT_COUNT"] == "0":
+        # installing kernel package
+        assert shell("yum install kernel-3.10.0-1160.el7.x86_64 -y").returncode == 0
+        # set default kernel
+        if "centos-7" in SYSTEM_RELEASE_ENV:
+            assert shell("grub2-set-default 'CentOS Linux (3.10.0-1160.el7.x86_64) 7 (Core)'").returncode == 0
+        elif "oracle-7" in SYSTEM_RELEASE_ENV:
+            assert (
+                shell("grub2-set-default 'Oracle Linux Server 7.9, with Linux 3.10.0-1160.el7.x86_64'").returncode == 0
+            )
+
+        # replace url in yum.repos.d rhel repo
+        original_url = (
+            r"baseurl = http://rhsm-pulp.corp.redhat.com/content/dist/rhel/server/7/$releasever/$basearch/os/"
+        )
+        new_url = "baseurl=http://rhsm-pulp.corp.redhat.com/content/dist/rhel/server/7/7.9/x86_64/os/"
+        shell('sed -i "s+{}+{}+g" /etc/yum.repos.d/rhel7.repo'.format(original_url, new_url))
+        shell("tmt-reboot -t 600")
+
+    if os.environ["TMT_REBOOT_COUNT"] == "1":
+        # remove all kernels except the kernel-3.10.0-1160.el7.x86_64
+        assert shell("rpm -qa kernel | grep -v 'kernel-3.10.0-1160.el7.x86_64' | xargs yum -y remove")
+        assert shell("rpm -qa kernel | wc -l").output == "1\n"
+        shell("tmt-reboot -t 600")
+
+
+@pytest.mark.test_one_kernel_scenario
+def test_one_kernel_scenario(shell, convert2rhel, one_kernel):
+    """TODO(r0x0d) better description and function name"""
```

Comment:
where description? :)

Reply:
<insert borek emoji here>

Reply:
You can probably link this issue there: https://issues.redhat.com/browse/RHELC-922. I'm not sure about the test anymore. We can probably revisit it after the new year and decide what to do with this one.